changelog.md

```diff
@@ -1,5 +1,10 @@
 # Express Routes Controllers logs
 
+## 0.3.3
```

Comment:
we are updating the node version, we should change the middle version, what do you think?

Reply:
as you prefer, in this case it's not a major change, but we can change it. I'll update in a few.

Reply:
updated!